test/sequential/sequencerStaking.rules.test.ts

```diff
@@ -360,9 +367,16 @@ describe("sequencerStaking", () => {
     const eventFiltered = cancelResolution.filter(
       (x) => x.method === "L1ReadStored",
     );
+    expect(eventFiltered[0].event.data[0].toHuman()).toContain(chain);
     expect(eventFiltered[0].event.data[1].toHuman()).toContain(
       preSetupSequencers.Ethereum,
     );
+    expect(JSON.parse(eventFiltered[0].event.data[3].toString()).start).toEqual(
+      txIndex,
+    );
+    expect(JSON.parse(eventFiltered[0].event.data[3].toString()).end).toEqual(
```

Comment:
what happens with (eventFiltered[0].event.data[2] ?

Reply:
![image](https://github.com/user-attachments/assets/a2be7a7f-fa5f-4c42-98a7-d65b568006b1)
should it not be block when executed?

Reply:
ITs tested elsewhere, closing.